### DISCUSSION TITLE
My merged PR was not in the latest release

### DISCUSSION THREAD

**Discussion by ghost** [Shopify/maintenance_tasks#668] [Q&A] [answered]
Curious about the release process of this project. I had a merged [PR](https://github.com/Shopify/maintenance_tasks/pull/653) but it wasn't included in the latest v1.10.2 release and it would have improved developer happiness.

**Comment by nvasilevski** [comment, marked as answer]
Hey @adi-landing, sorry about that, I thought that the latest released was supposed to only include `tick_count` migration - https://github.com/Shopify/maintenance_tasks/pull/663
But I'll need to double check with the team to make sure we are on the same page regarding releases

Personally I see no reason not to release your contribution so I'll make sure it's included in the next release. Perhaps it was forgotten by an accident

**Comment by etiennebarrie** [comment]
I limited the PRs I shipped to bug fixes. Maybe I should have considered yours a bug fix as well, but it didn't really seem like it, more like an enhancement.
We should really aim at the 2.0 anyway for the new features and with the breaking changes.
